mlflow/models/model.py

```diff
@@ -22,12 +23,37 @@
     "Logging model metadata to the tracking server has failed, possibly due older "
     "server version. The model artifacts have been logged successfully under %s. "
     "In addition to exporting model artifacts, MLflow clients 1.7.0 and above "
-    "attempt to record model metadata to the  tracking store. If logging to a "
-    "mlflow server via REST, consider  upgrading the server version to MLflow "
+    "attempt to record model metadata to the tracking store. If logging to a "
+    "mlflow server via REST, consider upgrading the server version to MLflow "
     "1.7.0 or above."
 )
 
 
+class ModelInfo(NamedTuple):
+    run_id: str
+    artifact_path: str
+    model_uri: str
+    utc_time_created: str
+    flavors: Dict[str, Any]
+    model_uuid: str
+    saved_input_example_info: Dict[str, Any]
+    signature: ModelSignature
```

Comment:
```suggestion
    signature: Optional[ModelSignature]
```

Can we use `Optional` for properties that can be None?

Reply:
I changed it to a native python dictionary that is `ModelSignature.to_dict()` to avoid importing pandas in mlflow skinny. Another reason is that for the user's information, there is no need to return the `ModelSignature` object. A dictionary should be sufficient for the purpose of information reading.